mne_connectivity/effective.py

```diff
@@ -12,6 +12,7 @@
 from .spectral import spectral_connectivity
 
 
+@verbose
```

Comment:
Looks like you're missing an import for this. There was also more than one place that this is needed, I only pointed out one example, check the diff